azure-cli.pyproj

```diff
@@ -394,9 +394,6 @@
     </Compile>
     <Compile Include="command_modules\azure-cli-redis\azure\cli\command_modules\redis\_params.py" />
     <Compile Include="command_modules\azure-cli-redis\azure\cli\command_modules\redis\__init__.py" />
-    <Compile Include="command_modules\azure-cli-redis\azure\cli\command_modules\__init__.py" />
-    <Compile Include="command_modules\azure-cli-redis\azure\cli\__init__.py" />
```

Comment:
Yes. They are excluded from the VS project in the other modules (though I found a few more that had them!)

---
In reply to: [77390429](https://github.com/Azure/azure-cli/pull/804#discussion_r77390429) [](ancestors = 77390429)